imagestorebackupstorage/ansible/imagestorebackupstorage.py

```
@@ -72,6 +72,7 @@
 IS_AARCH64 = host_info.host_arch == 'aarch64'
 IS_MIPS64EL = host_info.host_arch == 'mips64el'
 IS_LOONGARCH64 = host_info.host_arch == 'loongarch64'
+IS_SW64 = host_arch == 'sw_64'
```

Ruff (0.8.0)

`host_arch` may be undefined, or defined from star imports

(F405)